### PR TITLE
[feat] integrate SEM autofocusing in the pre-calibrations

### PR DESCRIPTION
**Pull request by tepals**
SEM autofocus has been added to fastem-calibrations and is now integrated in the pre-calibrations.
* A constant has been added to define how often to run SEM autofocusing (every n ROAs).
* The SEM autofocus button in the GUI now points to the new autofocusing ** For SEM autofocus and autocontrastbrightness also disable the button of the autofunction that is running, as these cannot be cancelled

### CODE REVIEW

File: src/odemis/gui/cont/acquisition/fastem_acq.py [hunk 1]
```diff
@@ -532,25 +532,34 @@ def on_acquisition(self, evt):
...
+                      f"and autofocus every {autofocus_period} sections.")
 
         for p in self._tab_data_model.projects.value:
             for idx, roa in enumerate(p.roas.value):
-                if idx == 0 or idx % autostig_period != 0:
-                    pre_calib = pre_calibrations
+                pre_calib = pre_calibrations.copy()
+                if idx == 0:
+                    pass
                 else:
```
Comment (pieleric) [inline src/odemis/gui/cont/acquisition/fastem_acq.py:558]: I think here would be clearer to explicitly write:
```suggestion
                if idx > 0:
```

File: src/odemis/gui/conf/file.py [hunk 2]
```diff
@@ -245,6 +245,7 @@ def __init__(self):
         self.default.set("acquisition", "fn_count", "0")
         self.default.set("acquisition", "overlap", "0.06")
         self.default.set("acquisition", "autostig_period", "5")
+        self.default.set("acquisition", "autofocus_period", "5")
```
Comment (pieleric) [inline src/odemis/gui/conf/file.py:248]: Could you indicate the unit here in comment (and do the same for the autostig_period). It's in "number of RoA acquisitions", right?

**Review by pieleric — CHANGES_REQUESTED** [review]

**Review by pieleric — APPROVED** [review]

**Comment by nandishjpatel** [comment]
@tepals can you add the `se_detector` and `ebeam_focus` components in the unittest [here](https://github.com/tepals/odemis/blob/4b618088a40e11ba322c68bcb1a8501dcd881c66/src/odemis/acq/align/test/fastem_test.py#L75)

**Comment by tepals** [comment]
> @tepals can you add the `se_detector` and `ebeam_focus` components in the unittest [here](https://github.com/tepals/odemis/blob/4b618088a40e11ba322c68bcb1a8501dcd881c66/src/odemis/acq/align/test/fastem_test.py#L75)?

fixed

**Review by nandishjpatel — APPROVED** [review]

**Pull request merged**